### ISSUE TITLE
Upgrade xgettext to 0.20.2

### ISSUE THREAD

**Issue by dpy013** [nvaccess/nvda#11142]
based on
[Upgrade xgettext to 0.20.2 nvaccess/nvda-misc-deps](https://github.com/nvaccess/nvda-misc-deps/pull/19)
hello
gettext 0.20.2 was officially released six days ago.
[Here is the change log for this version](http://savannah.gnu.org/forum/forum.php?forum_id=9716)
Update gettext0.18.1.0 to 0.20.2 from [mlocati gettext-iconv-windows](https://github.com/mlocati/gettext-iconv-windows)
thanks

**Comment by dingpengyu** [comment]
hello
I used xgettext 0.20.2 to build the NVDA in the local environment, and the translation was normal. The .mo was generated from the .po.
test environment
Visual Studio 2019 version16.5.4
python 3.7.7
scons 3.1.2
windows10-2004 19041.207
virtualenv 20.0.20
thanks

**Comment by bhavyashah** [comment]
@dingpengyu  Are there any key improvements in this new version as it relates to NVDA? If there are, then this ticket should be assigned a suitable priority and be given due attention.

**Comment by Adriani90** [comment]
cc: @feerrenrut maybe relevant for maintenance.
